src/components/card/card.styles.tsx

```diff
@@ -0,0 +1,91 @@
+import styled, { css } from "styled-components";
+import { ICardContentStyles } from "./types";
+import { DragHandleIcon } from "@lifesg/react-icons/drag-handle";
+import { Button } from "@lifesg/react-design-system/button";
+import { Color } from "@lifesg/react-design-system/color";
+import { Spacer } from "../common/spacer";
+
+export const CardContent = styled.div<ICardContentStyles>`
```

Comment:
for the different states, I notice it differs from [figma](https://www.figma.com/file/G9h7ux7jd3IGnJn6OEfDhC/%5BWeb%5D-Sequential?type=design&node-id=3903-50805&mode=design&t=KatRg30j2NGo692p-0), e.g. on hover, the box shadow is in primary colour, the dragger is missing, duplicate and delete colours are different from the figma

Reply:
Due to the blockage for other parts, we will do the dragger in another ticket